apostrophecms/apostrophe
# Pro 2282 pro 2236 import button and modal

[PRO-2182](https://linear.app/apostrophecms/issue/PRO-2182/as-an-editor-i-can-click-a-button-to-open-a-modal-for-importing-pieces)
[PRO-2236](https://linear.app/apostrophecms/issue/PRO-2236/as-an-editor-i-am-presented-with-a-modal-screen-to-upload-my-file-for)

## Purpose
Adds a new component `AposFile.vue` to abstract file dropzone UI. Uses this new component in `AposInputAttachment`. And uses it in the pieces import modal.

## How to test
Check that normal attachment UI is still working as expected.
Test the new import modal by trying to import csv files. (confirm button does nothing for now).
Import can be test with y linking this module's PR: https://github.com/apostrophecms/piece-type-importer/pull/1
And the `testbed` project has an `import` branch with this new module configured.
